code_samples/front/render_content/templates/themes/my_theme/pagelayout.html.twig

```diff
@@ -0,0 +1,5 @@
+<head>{{ encore_entry_link_tags('style') }}</head>
+
+{% block content %}
+{% endblock %}
+<footer></footer>
```

Comment:
Missing newline will make `git` unhappy when adding new lines at the end :smile: